common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -173,3 +173,12 @@ class VideoFields(object):
         scope=Scope.settings,
         default="",
     )
+    date_last_view_bumper = DateTime(
+        display_name=_("Date of the last view of the bumper"),
+        scope=Scope.preferences,
+    )
+    do_not_show_again_bumper = Boolean(
```

Comment:
@auraz Umm, I was thinking more like `bumper_show_again` would be `True` by default and once the user clicks "skipsAndDoNotShowAgain" we would set this to `False`. It was just a small readability suggestion, it is fine as it is. :)

It would still be nice to change `date_last_view_bumper` to `bumper_last_view_date `.